src/esp/gfx/Viewer.cpp

```diff
@@ -126,6 +136,13 @@ Viewer::Viewer(const Arguments& arguments)
   renderCamera_->node().setTransformation(
       cameraNode_->absoluteTransformation());
 
+#ifndef ESP_BUILD_WITH_BULLET
+  if (enablePhysics_)
+    LOG(WARNING) << "Physics was enabled but the project is built without "
+                    "Bullet support, objects added with the O key won't "
+                    "interact with the environment in any way.";
+#endif
```

Comment:
Note that one use case is to define "physics simulator": "none" in default.phys_scene_config.json to enable kinematic objects only. This is also the default behavior when "bullet" is set in config and Habitat-sim was not installed with Bullet. In this case [ResourceManager line 136](https://github.com/facebookresearch/habitat-sim/blob/be178edd9a5e71fb5964631748597c6ac6668b2f/src/esp/assets/ResourceManager.cpp#L136) prints an error). Maybe we should modify that text to be more precise instead of placing another warning here to avoid warning when the user intends for kinematics?

Reply:
You know better than me what's the proper place where to put this and what should it say, so feel free to push commits on top of this -- I promise to not force-push :)

In my case I didn't get *any* warning when I simply used the default-provided files with default settings, so that's why I put it here.